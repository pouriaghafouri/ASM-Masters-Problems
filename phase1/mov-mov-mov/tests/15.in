-4008211666962689380 1817615580039675579
