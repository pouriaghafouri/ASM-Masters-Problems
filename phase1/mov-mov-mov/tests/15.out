-5825827247002364959
